addons/l10n_ma/views/res_company_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="res_company_form_l10n_ma_view_company" model="ir.ui.view">
+        <field name="name">res.company.form.inherit.l10n_ma</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="account.view_company_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='company_registry']" position="after">
+                <field name="l10n_ma_ice" invisible="country_code != 'MA'"/>
+            </xpath>
+            <xpath expr="//field[@name='company_registry']" position="attributes">
+                <attribute name="invisible">country_code == 'MA'</attribute>
```

Comment:
Has this been approved by the PO ? I don't see anything about removing the company registry in the task. Just wondering :thinking:

Reply:
Yes, It was a question raised before and we agreed the the ICE should be a replacement of the company_registry.